src/components/Requirements.vue

```diff
@@ -265,6 +272,12 @@ export default {
       ],
       requirementsMap: {
         // CS 1110: 'MQR-AS'
+      },
+      // reqGroupColorMap maps reqGroup to an array [<hex color for progress bar>, <color for arrow image>]
+      reqGroupColorMap: {
+        UNIVERSITY: ['92C3E6', 'lightblue'],
+        COLLEGE: ['1AA9A5', 'blue'],
+        MAJOR: ['105351', 'green']
```

Comment:
Maybe add a minor color in here for longevity? If Shane didn't design one, pm him.

Reply:
In which version of the designs are the color schemes for those req groups? I made a commit with the colors from the 'Sidebar' board on Figma

Reply:
I messaged Shane about the color for University but he didn't give me a color yet. I think for now the courseplan logo's fourth unused color could work!